pkg/client/unversioned/clientcmd/client_config.go

```diff
@@ -88,6 +88,21 @@ func (config *DirectClientConfig) RawConfig() (clientcmdapi.Config, error) {
 	return config.config, nil
 }
 
+// cleanHost cleans URI, checks if any opaque data is present and
+// path is longer than 1 symbol.
+func cleanHost(host string) (string, error) {
+	u, err := url.ParseRequestURI(host)
+	if err != nil {
+		return "", err
+	}
+	if u.Opaque != "" && len(u.Path) <= 1 {
```

Comment:
Can you add a comment here (or somewhere around this function) with some example values that you are trying to filter out